tests/scripts/remote_monitoring_tests/helpers/utils.py

```diff
@@ -189,8 +189,6 @@ def generate_test_data(csvfile, test_data):
 
                 test_name = t + "_" + key
                 status_code = 400
```

Comment:
We need to include the below code back and pass createExperiment or updateResults as a parameter to generate_test_data and include the below code for createExp
```
 if test_name == "invalid_experiment_name" or test_name == "invalid_cluster_name":
                    status_code = 201
```

Reply:
Also include validation for the expected failure messages

Reply:
> We need to include the below code back and pass createExperiment or updateResults as a parameter to generate_test_data and include the below code for createExp
> 
> ```
>  if test_name == "invalid_experiment_name" or test_name == "invalid_cluster_name":
>                     status_code = 201
> ```

This is done. Validation for the failure messages will be taken up in a different PR.